Tests/service-test.yml

```diff
@@ -0,0 +1,65 @@
+config:
+  target: "http://localhost:4006"
+  http:
+    timeout: 10
```

Comment:
Also didn't know about the timeout variable for your config - also pretty cool, very nice